css/dashboard.css

```diff
@@ -136,6 +159,15 @@ main .card {
   line-height: 19px;
   color: #222222;
   font-weight: 700;
+  padding: 0.75rem;
+}
+
+.thead-light th {
+  background-color: #f8f8f8 !important;
+  border: none !important;
+  color: #222 !important;
+  font-size: 14px !important;
+  line-height: 16px !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5469161)